Automattic/jetpack
# PWA: Adds lazy image loading module

Will help speed by minimizing the number of images we download when the page is first viewed.

This plugin is largely based on the [Lazy Load](https://github.com/Automattic/lazy-load) plugin, but has been updated to make use of IntersectionObserver. IntersectionObserver does not have great browser support. But, we are using a polyfill that will add the necessary functionality.

I have several questions, which are listed in [this comment](https://github.com/Automattic/jetpack/pull/8093#issuecomment-342267044), mostly around attribution and how we should load the polyfill. 

Besides that, this PR should be ready for some review and testing.

To test:

- Checkout PR
- Load admin page
- Go to `$site.com/wp-admin/admin.php?page=jetpack_modules` and enable the Lazy Images module
- Go to front-end of site, and ensure that images get loaded as you scroll down the page. Tip: you can filter the network panel to only show img requests
- Ensure that there are no JS errors
- Test various browsers
